### PR TITLE
Clean options

### PR DESCRIPTION
**Pull request by ajreynol**
Deletes many unnecessary options, cleans some documentation, consolidates several options.

### CODE REVIEW

File: src/theory/quantifiers/quant_conflict_find.cpp [hunk 1]
```diff
@@ -2045,15 +2080,15 @@ bool MatchGen::doMatching()
...
-    switch( typ ){
-    case typ_invalid: Trace(c) << "invalid";break;
-    case typ_ground: Trace(c) << "ground";break;
-    case typ_eq: Trace(c) << "eq";break;
-    case typ_pred: Trace(c) << "pred";break;
-    case typ_formula: Trace(c) << "formula";break;
-    case typ_var: Trace(c) << "var";break;
-    case typ_bool_var: Trace(c) << "bool_var";break;
-    }
+  switch( typ ){
```
Comment (4tXJ7f) [inline src/theory/quantifiers/quant_conflict_find.cpp:2083]: ClangFormat somehow only partially formatted this?

**Review by 4tXJ7f — CHANGES_REQUESTED** [review]
Just some minor comments.

**Comment by ajreynol** [comment]
Formatting issues should be fixed now.

**Review by 4tXJ7f — APPROVED** [review]
LGTM, thanks for the updates.

**Pull request merged**